src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -166,149 +141,113 @@ export default class rd2EntryForm extends LightningElement {
     * @description Construct field describe info from the Recurring Donation SObject info
-    */ 
+    */
     setFields(fieldInfos) {
         this.fields.name = this.extractFieldInfo(fieldInfos[FIELD_NAME.fieldApiName]);
         this.fields.account = this.extractFieldInfo(fieldInfos[FIELD_ACCOUNT.fieldApiName]);
         this.fields.contact = this.extractFieldInfo(fieldInfos[FIELD_CONTACT.fieldApiName]);
-        this.fields.recurringType = this.extractFieldInfo(fieldInfos[FIELD_RECURRING_TYPE.fieldApiName]);
         this.fields.dateEstablished = this.extractFieldInfo(fieldInfos[FIELD_DATE_ESTABLISHED.fieldApiName]);
-        this.fields.amount = this.extractFieldInfo(fieldInfos[FIELD_AMOUNT.fieldApiName]);
-        this.fields.paymentMethod = this.extractFieldInfo(fieldInfos[FIELD_PAYMENT_METHOD.fieldApiName]);
-        this.fields.period = this.extractFieldInfo(fieldInfos[FIELD_INSTALLMENT_PERIOD.fieldApiName]);
-        this.fields.installmentFrequency = this.extractFieldInfo(fieldInfos[FIELD_INSTALLMENT_FREQUENCY.fieldApiName]);
-        this.fields.dayOfMonth = this.extractFieldInfo(fieldInfos[FIELD_DAY_OF_MONTH.fieldApiName]);
-        this.fields.startDate = this.extractFieldInfo(fieldInfos[FIELD_START_DATE.fieldApiName]);
         this.fields.campaign = this.extractFieldInfo(fieldInfos[FIELD_CAMPAIGN.fieldApiName]);
-        this.fields.currency = {label: currencyFieldLabel, apiName: 'CurrencyIsoCode'};
-        this.fields.plannedInstallments = this.extractFieldInfo(fieldInfos[FIELD_PLANNED_INSTALLMENTS.fieldApiName]);
     }
 
-    /*******************************************************************************
-    * @description Method converts field describe info into a object that is easily accessiable from the front end
+    /***
+    * @description Method converts field describe info into a object that is easily accessible from the front end
     */
     extractFieldInfo(field) {
         return {
-            apiName : field.apiName,
-            label : field.label,
-            inlineHelpText : field.inlineHelpText,
-            dataType : field.dataType
+            apiName: field.apiName,
+            label: field.label,
+            inlineHelpText: field.inlineHelpText,
+            dataType: field.dataType
         };
     }
 
-    /*******************************************************************************
+    /***
     * @description Retrieve Recurring Donation record value
-    */ 
+    */
     @wire(getRecord, { recordId: '$recordId', fields: '$fieldInfos' })
     wiredRecurringDonationRecord(response) {
         if (response.data) {
             this.record = response.data;
-            this.modifyEditHeader();
+            this.header = editHeaderLabel + ' ' + this.record.fields.Name.value;
             this.isRecordReady = true;
         }
 
         if (response.error) {
             this.isRecordReady = true;
-            handleError(response.error);
+            this.handleError(response.error);
```

Comment:
The current code in `feature/RD2` uses `handleError` methods without specifying `this.` in few places, so that can be corrected in another PR. @andrewyu-salesforce , @force2b